.azure-pipelines/build-template.yml

```diff
@@ -92,29 +93,29 @@ jobs:
   - task: DownloadPipelineArtifact@2
     inputs:
       source: specific
-      project: build
-      pipeline: Azure.sonic-sairedis
-      artifact: ${{ parameters.sairedis_artifact_name }}
+      project: nvidia
```

Comment:
Looks like this project is not accessible from `mssonic`: https://dev.azure.com/mssonic/build/_build/results?buildId=165280&view=logs&jobId=83516c17-6666-5250-abde-63983ce72a49&j=18fdc12d-bdfb-52d7-1391-aaa772f0f664

Reply:
Maybe it's better to add new steps for downloading the the DASH-specific DVS/libraries? This way when we eventually merge to master it won't interfere with the regular DVS image build

Reply:
Fixed in another PR: https://github.com/sonic-net/sonic-swss/pull/2501